app/assets/stylesheets/components/_progress.scss

```diff
@@ -0,0 +1,70 @@
+//--------------------------------------------------
+// variables
+//--------------------------------------------------
+
+//--------------------------------------------------
+// mixins
+//--------------------------------------------------
+
+//--------------------------------------------------
+// classes
+//--------------------------------------------------
+.progress {
+  @include flex;
+  @include flex-v-center;
+  width: 100%;
+  background-color: $grey-mdlight;
+  font-weight: bold;
+  list-style: none;
+  margin: 0 2rem;
+  padding: 0;
+
+  &__active {
+    background-color: $green-dark;
+    color: $white;
+    padding: 2rem;
+
+    &:after {
+      width: 0; height: 0;
+      margin-top: -2rem;
+      margin-left: 2rem;
+      z-index: 1;
+
+      content: '';
+      position: absolute;
+
+      border-style: solid;
+      border-width: 2.563rem 0 2.563rem 3.438rem;
+      border-color: transparent transparent transparent $green-dark;
```

Comment:
I think the order/spacing here should be:

```
border-color: transparent transparent transparent $green-dark;
border-style: solid;
border-width: 2.563rem 0 2.563rem 3.438rem;
content: '';
margin-top: -2rem;
margin-left: 2rem;
width: 0; height: 0;

position: absolute;
z-index: 1;
```

Reply:
oh ok, sorry, worked mainly with tailwind so far 😅